list/list.css

```diff
@@ -0,0 +1,94 @@
+* {
+    font-family: 'Nunito Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.component-title {
+    font-size: 1.8rem;
+    font-weight: bold;
+    margin-bottom: 1.5rem;
+}
+
+.component-description {
+    font-size: 1.1rem;
+    margin-bottom: 1.5rem;
+}
+
+.side-heading {
+    font-size: 1.4rem;
+    font-weight: bold;
+    margin-bottom: 1.5rem;
+}
+
+.side-heading-description {
+    font-size: 1.1rem;
+    margin-bottom: 1.5rem;
+}
+
+
+/* ordered list */
+
+.lists {
+    display: flex;
+    justify-content: center;
+}
+
+
+/* spaced list */
+
+.spaced-list-items {
+    list-style: none;
+    margin-bottom: 1.1rem;
+    padding: 1rem 1rem;
+    text-align: center;
+    background-color: #f1f1f1;
```

Comment:
You are using the same background color for stacked-list-item and nested-list as well, I would suggest you to use a variable.

Reply:
> You are using the same background color for stacked-list-item and nested-list as well, I would suggest you to use a variable.

sure saran will work on it